templates/include/glean/glean.1.schema.json

```diff
@@ -26,6 +26,7 @@
         "first_run_date": @GLEAN_DATETIME_1_JSON@,
         "os": @GLEAN_STRING_1_JSON@,
         "os_version": @GLEAN_STRING_1_JSON@,
+        "android_sdk_version": @GLEAN_STRING_1_JSON@,
```

Comment:
Strictly speaking, this is always an integer, right?

I don't think there's much value in enforcing that at this level, just wanted to flag it in case there might be...

Reply:
Good call. I'll make this an int.

Reply:
Actually, as we discussed on Slack, we don't have the "number" plain type in glean, so.. let it be a string :)